src/components/Toggle.vue

```diff
@@ -15,24 +15,26 @@ export default defineComponent({
      * the prop modelValue is required to use [v-model](https://vuejs.org/guide/components/events.html#usage-with-v-model) with a component
      */
     modelValue: {
-      type: Boolean,
+      type: [Boolean, String] as PropType<boolean | 'mixed'>,
```

Comment:
is there a reason why it's different from `Checkbox`?

```
    /**
     * the prop modelValue is required to use [v-model](https://vuejs.org/guide/components/events.html#usage-with-v-model) with a component
     */
    modelValue: {
      type: [Boolean, Array],
      required: true
    },
    /**
     * [indeterminate](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes) state for checkbox,
     * e.g. a select/deselect all checkbox in a table
     */
    mixed: {
      type: Boolean,
      default: false
    },
```

Reply:
it is different because checkbox's `indeterminate` property is different from `aria-pressed` attribute of a toggle button.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#indeterminate_state_checkboxes
the docs say `indererminate` on an `<input of type="checkbox/>` element cannot be set via an HTML attribute, but only via JS as a property on the element. As far as I understand this doesn't change the input's `checked` attribute.
 
 On the other hand, the `aria-pressed` attribute supports having a boolean value or a `mixed` value, and it is has one attribute that handles both cases, not a separate flag like with checkboxes.
 
 We _could_ smooth that out internally and expose a consistent external api, but I didn't had that goal in mind while making this change, and I'm not even sure it's worth it in general (ie. there can be arguments against obscuring native HTML apis)
 
 PS. I can't remember why the checkbox accepts an array value. not sure it it was intentional or some kind of bad copy-paste no one noticed. worth looking into